app/src/main/java/org/piramalswasthya/sakhi/repositories/AbhaIdRepo.kt

```
@@ -250,6 +251,30 @@ class AbhaIdRepo @Inject constructor(
+    suspend fun printAbhaCard(): NetworkResult<String> {
+        return withContext(Dispatchers.IO) {
+            try {
+                val response = abhaApiService.printAbhaCard(generateUUID(), getCurrentTimestamp())
+                val responseBody = response.body()?.bytes()
+                if (response.isSuccessful) {
+                    NetworkResult.Success(
+                        responseBody.toString())
+                } else {
+                    sendErrorResponse(response)
+                }
+            } catch (e: IOException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)